Userland/Libraries/LibC/wstdio.cpp

```diff
@@ -207,6 +207,11 @@ int vswprintf(wchar_t* __restrict wcs, size_t max_length, wchar_t const* __restr
         ++length_so_far;
     },
         wcs, fmt, args);
+    if (length_so_far < max_length) {
+        wcs[length_so_far] = L'\0';
+    } else {
+        wcs[max_length - 1] = L'\0';
+    }
```

Comment:
We generally don't put curly braces around single-line if/else statements

Reply:
Thank you Tim. I pushed new commit changing the message as well as braces.